src/lib.rs

```diff
@@ -40,30 +40,32 @@ extern crate byteorder;
 extern crate num_traits;
 extern crate serde as serde_crate;
 
-mod serde;
+mod ser;
+mod de;
+pub mod internal;
```

Comment:
I renamed `endian_choice` to `internal` and kept it public.  I'm debating if I should switch back to the old name though.